src/components/Reuseable-Components/Scenes/CraftModal.js

```diff
@@ -0,0 +1,250 @@
+import React, { useState } from 'react';
```

Comment:
Looks good as well, but we probably need to make a new repo for React components and then include this in there. The Home docs are really meant to just be the place we document our components, if that makes sense.